plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/tasks/AbstractGenerateClientTask.kt

```diff
@@ -66,7 +66,7 @@ abstract class AbstractGenerateClientTask : DefaultTask() {
      */
     @InputFile
     @Optional
-    val schemaFile: RegularFileProperty = project.objects.fileProperty()
+    val schemaFile: Property<String> = project.objects.property(String::class.java)
```

Comment:
there already is `schemaFileName` property for that use case, this should remain a `RegularFileProperty`